bin/crew

```diff
@@ -1485,33 +1485,37 @@ def upload(pkg_name = nil, pkg_version = nil, gitlab_token = nil, gitlab_token_u
         end
       end
       puts "Package: #{package}, Arch: #{arch}".yellow
-      puts 'Generating sha256sum ...'
+      puts
+      puts "\e[1A\e[KGenerating sha256sum ...\r".orange
```

Comment:
Isn't `\r` only a Macism?

Reply:
I know, are you running your Docker containers on a Mac?

Reply:
Nope.